SiraUtil/Tools/FPFC/FPFCToggle.cs

```diff
@@ -180,7 +181,24 @@ private void DisableFPFC()
             foreach (var listener in _fpfcListeners)
                 listener.Disabled();
 
-            InitializeXRLoader();
+            _siraLog.Debug(IsProcessRunning("vrserver") ? "VR Server is running" : "VR Server is not running");
+            _siraLog.Debug(IsProcessRunning("vrcompositor") ? "VR Compositor is running" : "VR Compositor is not running");
+            _siraLog.Debug(Environment.GetCommandLineArgs().Any(s => s.ToLower() == "oculus") ? "Running as vrmode Oculus" : "Running as vrmode Steam");
+            bool startXR = (IsProcessRunning("vrserver") && IsProcessRunning("vrcompositor")) || Environment.GetCommandLineArgs().Any(s => s.ToLower() == "oculus"); // If we're running Oculus, we need to start XR
+            if (startXR) InitializeXRLoader();
```

Comment:
Can you move the method to the next line, I find same line if statements to be more unreadable.
```
if (...)
   Thing();
```